### PR TITLE
DX-2411 Add a warning about expiring cookies

### PR DESCRIPTION
**Pull request by kirillsud**

### CODE REVIEW

File: examples/nextjs-full-stack/README.md [hunk 1]
```diff
@@ -60,6 +60,12 @@ Once your server is up and running:
 2. Open a board in your Developer team.
 3. To start the app, click the corresponding app icon in the app toolbar on the left.
 
+> ⚠️ In case of error `401` returned by a REST API request, remove the `miro_tokens` cookie for the `localhsot`
+> domain in the browser.
+>
```
Comment (horeaporutiu) [inline examples/nextjs-full-stack/README.md:65]: Is it possible to add either a screenshot / explanation of how to do that? I'm still not sure myself, so I think some devs maybe will not how to remove cookies for localhost specifically.

*typo in `localhsot`

Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:65]: Is it 

Chrome -> Settings -> Privacy and Security -> Clear browsing data? I think this is not right.

Comment (kirillsud) [reply examples/nextjs-full-stack/README.md:65]: Well, it depends on the browser. For example, we can add a screenshot for Chrome's DevTools:

image

Comment (kirillsud) [reply examples/nextjs-full-stack/README.md:67]: Added

Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:67]: looks great to me!

**Review by horeaporutiu — APPROVED** [review]

File: examples/nextjs-full-stack/README.md (hunk 1 above)
Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:67]: Actually one small typo for `localhost` in line 63 @kirillsud

Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:67]: Screenshot 2023-09-14 at 13 31 38

What is a bit strange is I dont have any cookies for localhost and I still see the 401 error

Comment (horeaporutiu) [reply examples/nextjs-full-stack/README.md:67]: sorry please ignore now I do see them

**Review by janza — APPROVED** [review]

**Comment by mettin** [comment]
Should we have this in create-react-app too?

**Pull request merged**